clang/lib/Driver/Driver.cpp

```diff
@@ -802,6 +802,18 @@ static bool addSYCLDefaultTriple(Compilation &C,
   return true;
 }
 
+// Prefix for Intel GPU specific targets used for -fsycl-targets
+constexpr char IntelGPU[] = "intel_gpu_";
+
+static bool isIntelGPUTarget(StringRef Target, StringRef &Device) {
```

Comment:
Not fundamental but at the end it looks like it would more modern to return an StringRef optional.